src/shared_modules/content_manager/src/components/offlineDownloader.hpp

```diff
@@ -133,7 +185,15 @@ class OfflineDownloader final : public AbstractHandler<std::shared_ptr<UpdaterCo
 
 public:
     /**
-     * @brief Copies a file from the local filesystem in order to be processed and passes the control to the next chain
+     * @brief Class constructor.
+     *
+     * @param urlRequest HTTP driver instance to use within the class.
+     */
+    explicit OfflineDownloader(IURLRequest& urlRequest)
+        : m_urlRequest(urlRequest) {};
+
+    /**
+     * @brief Downloads a file in offline mode in order to be processed and passes the control to the next chain
```

Comment:
Question: Shouldn't this express that the file could be obtained from the local filesystem or from an HTTP/HTTPS server?

Reply:
That's what I wanted to express by just saying "offline mode". For a more detailed explanation, there is the documentation.